vital/algo/detected_object_set_output.cxx

```diff
@@ -18,17 +18,19 @@
 #include <kwiversys/SystemTools.hxx>
 
 /// \cond DoxygenSuppress
-INSTANTIATE_ALGORITHM_DEF(kwiver::vital::algo::detected_object_set_output);
+INSTANTIATE_ALGORITHM_DEF( kwiver::vital::algo::detected_object_set_output );
 /// \endcond
 
 namespace kwiver {
+
 namespace vital {
+
 namespace algo {
 
 detected_object_set_output
 ::detected_object_set_output()
   : m_stream( 0 )
-  , m_stream_owned( false )
+    , m_stream_owned( false )
```

Comment:
This is somewhat atrocious. I doubt Uncrustify is correctly configured for this style, and anyway I'm not sure we should be encouraging it. (I also seem to recall Uncrustify is failing to move the `,` to the preceding line, despite theoretically being configured to do so; n.b. `pos_constr_comma`.)

Reply:
I grepped through and think I caught and fixed all these.